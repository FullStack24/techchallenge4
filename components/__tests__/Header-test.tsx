import * as React from 'react';
import renderer from 'react-test-renderer';
import Header from '../HeaderFooter/Header';

it('deve renderizar o header com o logo corretamente', () => {
  const tree = renderer.create(
    <Header />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
