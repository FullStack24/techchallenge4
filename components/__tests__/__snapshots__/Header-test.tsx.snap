// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`deve renderizar o header com o logo corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#000000",
      "padding": 16,
    }
  }
>
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    Bloguinho
  </Text>
</View>
`;
